app/controllers/users_controller.rb

```diff
@@ -235,6 +239,22 @@ def change_email(user_data)
     end
   end
 
+  def upload_export_files(user_data)
+    logger.info "Start importing profile"
+    @user.export = user_data[:export] if user_data[:export]
+    @user.exported_photos_file = user_data[:exported_photos_file] if user_data[:exported_photos_file]
+    if @user.save
+      flash.now[:notice] = "A profile migration is scheduled"
+    else
+      flash.now[:error] = "An error occured scheduling a migration: #{@user.errors.full_messages}"
```

Comment:
It looks as though a specific error message is appended to this warning, so I suggest:
- The following error occurred in scheduling your account migration:
or
- Your account migration could not be scheduled for the following reason:
(note: two Rs in occurred)